src/chapters/jdbc-auth.xml

```diff
@@ -805,6 +805,48 @@ postgresql-user-required: true</programlisting>
                             zones.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>full_name</property></term>
+                    <listitem>
+                        <para>The user's full name. Unlike the username, this name need not be
```

Comment:
Not that I think it's super needed, but why does this paragraph not have a thing about setting the column to NULL like the others?